tests/test_views.py

```diff
@@ -0,0 +1,1050 @@
+        response = views.project_about(request, project.project_slug)
+
+        assert response.status_code == 200
+
+    def test_template_used(self, client):
+        project = factories.ProjectFactory()
+        response = client.get(reverse('project_about', args=[project.project_slug]))
+        assert response.templates[0].name == 'nomination/project_about.html'
+
+    @pytest.mark.parametrize('nomination_end, show_bookmarklets', [
+        (datetime.now() - timedelta(days=1), False),
+        (datetime.now() + timedelta(days=1), True)
+    ])
+    def test_context(self, client, nomination_end, show_bookmarklets):
+        project = factories.ProjectFactory(nomination_end=nomination_end)
+        factories.URLFactory(url_project=project, attribute='surt')
+        factories.NominatedURLFactory.create_batch(2, url_project=project, value=1)
+        current_host = Site.objects.get(id=settings.SITE_ID)
+        response = client.get(reverse('project_about', args=[project.project_slug]))
+
+        assert response.context['project'] == project
+        assert response.context['url_count'] == 1
+        assert response.context['nominator_count'] == 2
+        assert response.context['current_host'] == current_host
+        assert response.context['show_bookmarklets'] == show_bookmarklets
+
+
+class TestBrowseJson():
+
+    def test_status_ok(self, rf):
+        project = factories.ProjectFactory()
+        request = rf.get('/')
+        response = views.browse_json(request, project.project_slug, '')
+
+        assert response.status_code == 200
+
+    def test_mimetype(self, rf):
+        project = factories.ProjectFactory()
+        request = rf.get('/')
+        response = views.browse_json(request, project.project_slug, '')
+
+        assert response['Content-Type'] == 'application/json'
+
+    @pytest.mark.parametrize('request_type, kwargs, id, text', [
+        ('get', {'root': 'com,'}, 'com,example,',
+         '<a href="surt/http://(com,example">com,example</a>'),
+        ('get', {'root': 'source'}, 'com,', 'com'),
+        ('get', {}, 'com,', 'com'),
+        ('post', {}, 'com,', 'com')
+    ])
+    def test_json_string(self, rf, request_type, kwargs, id, text):
+        project = factories.ProjectFactory()
+        factories.URLFactory(
+            url_project=project,
+            attribute='surt',
+            value='http://(com,example,www)'
+        )
+        request = getattr(rf, request_type)('/', kwargs)
+        response = views.browse_json(request, project.project_slug, '')
+
+        assert json.loads(response.content) == [{
+            'hasChildren': True,
+            'id': id,
+            'text': text
+        }]
+
+
+class TestSearchJson():
+
+    def test_status_ok(self, rf):
+        project = factories.ProjectFactory()
+        request = rf.get('/')
+        response = views.search_json(request, project.project_slug)
+
+        assert response.status_code == 200
+
+    def test_mimetype(self, rf):
```

Comment:
We can probably consolidate this into the test above. If not, it might be a good idea to change this to `test_content_type`.